activerecord/lib/active_record/connection_adapters/abstract/database_statements.rb

```diff
@@ -148,6 +148,25 @@ def exec_query(sql, name = "SQL", binds = [], prepare: false)
         internal_exec_query(sql, name, binds, prepare: prepare)
       end
 
+      # Convenience method to simplify running a SQL query with variables (binds)
+      def simple_query(sql, sql_vars = {}, **splat_sql_vars)
```

Comment:
1. I think `binds` or `variables` is a better name.
2. I also think this definition
```ruby
def simple_query(sql, binds)
```
covers all the cases - no need for a hash argument and a separate splat argument.

3. `simple_query` name can be confusing, but I don't have suggestions for a better one
4. The documentation for this method can be improved - see examples from the surrounding methods

Reply:
> def simple_query(sql, binds)
> covers all the cases - no need for a hash argument and a separate splat argument.

This syntax fails: `simple_query("SELECT id FROM posts WHERE id = :id", id: 1)`

I would like this method to behave the same as any normal AR method such as `where`

Reply:
Changed to argument name to `binds`